### PR TITLE
VIDEO-11537 Reverting to using import(), but within a Function.

### PR DESCRIPTION
**Pull request by manjeshbhargav**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### Description

Since we [don't want to depend on Blob](https://github.com/twilio/twilio-video.js/issues/1880) anymore, the dynamic import module will now revert to calling `import()` like before, but within a Function whose body is specified as a string, so TypeScript and Webpack (and other frameworks) should not have a problem with it. However, this change requires that any application using the CSP directive `default-src self` must also add the `unsafe-eval` directive, which is already required by the Krisp Noise Cancellation plugin.

### Before review
* [x] Updated CHANGELOG.md if necessary
* [ ] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with `npm test`
* [x] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [x] Ready for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +154,14 @@ you should also include the following `script-src` directive:
 script-src https://sdk.twilio.com
 ```
 
+If you are enabling [Krisp Noise Cancellation](https://www.twilio.com/docs/video/noise-cancellation) for
+your local audio, and you are using the following `default-src self` directive, you should also add the
+`unsafe-eval` directive:
+
+```
+default-src self unsafe-eval
+```
```
Comment (PikaJoyce) [inline README.md:163]:
```
default-src 'self' 'unsafe-eval';
```

File: lib/util/dynamicimport.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
```
Comment (PikaJoyce) [inline lib/util/dynamicimport.js:2]: I'm not sure if there should be a note here for us to come back around to fix this in the future? Might be useful for future us as a reminder.

**Review by PikaJoyce — APPROVED** [review]
Thanks Manjesh, this is a super awesome fix! 

I've also tested this locally and it works with those new directives! Left a few comments otherwise it LGTM 💯

File: README.md (hunk 1 above)
Comment (manjeshbhargav) [reply README.md:163]: What you've written is the client-side meta tag syntax. What I've specified is the server side syntax.

File: lib/util/dynamicimport.js (hunk 2 above)
Comment (manjeshbhargav) [reply lib/util/dynamicimport.js:2]: I've added a note above the line where we create the function.

**Pull request merged**